### DISCUSSION TITLE
Unauthenticated PHP mails

### DISCUSSION THREAD

**Discussion by serversupport** [rspamd/rspamd#3620] [Q&A]
Hi,

We have recently installed and configured rspamd on one of our DirectAdmin servers. rspamd detects all the incoming and outgoing mails; however, it does not detect unauthenticated mails and mails sent via scripts such as sendmail and php mail functions.

Is there any way on this such that the unauthenticated PHP mails could also be passed through the rspamd?

Thanks!

**Comment by andryyy** [comment]
It depends on the MTA configuration. Assuming you use Postfix you may have missed to setup milter for non-smtp connections.

**Reply by serversupport** [reply]
We are using exim as MTA.

**Reply by fatalbanana** [reply]
In this case you're likely looking for [acl_not_smtp](https://www.exim.org/exim-html-current/doc/html/spec_html/ch-access_control_lists.html) and/or it could be issue of what's contained in your ACLs (a message might be `accept`ed before scanning for e.g.).

**Reply by serversupport** [reply]
Thanks @fatalbanana 

That seems to have done the trick  :)
